apps/L1G/tinder/tinder-frontend/specs/ProfileImages.spec.tsx

```
@@ -0,0 +1,168 @@
+    const removeButton = screen.getByTestId('remove-button-0');
+    fireEvent.click(removeButton);
+
+    await waitFor(() => {
+      expect(screen.queryByTestId('uploaded-image-0')).not.toBeInTheDocument();
+    });
+  });
+
+  test('clears file input after upload', async () => {
+    const mockUrl = 'https://cloudinary.com/test-image.jpg';
+    (axios.post as jest.Mock).mockResolvedValueOnce({
+      data: { secure_url: mockUrl },
```

ESLint

[error] Identifier 'secure_url' is not in camel case.

(camelcase)